packages/devkit/src/utils/package-json.ts

```diff
@@ -150,15 +205,21 @@ function requiresAddingOfPackages(packageJsonFile, deps, devDeps): boolean {
   packageJsonFile.devDependencies = packageJsonFile.devDependencies || {};
 
   if (Object.keys(deps).length > 0) {
-    needsDepsUpdate = Object.keys(deps).some(
-      (entry) => !packageJsonFile.dependencies[entry]
-    );
+    needsDepsUpdate =
```

Comment:
As far as I see, this code will always return true in one of this cases, except when dep is provided both in devs and devDeps, which is impossible